app/products/calls/components/channel_info_enable_calls/index.tsx

```diff
@@ -28,6 +29,11 @@ const ChannelInfoEnableCalls = ({channelId, enabled}: Props) => {
     const disableText = formatMessage({id: 'mobile.calls_disable', defaultMessage: 'Disable calls'});
     const enableText = formatMessage({id: 'mobile.calls_enable', defaultMessage: 'Enable calls'});
 
+    const config = getCallsConfig(serverUrl);
```

Comment:
I think we should move this check into `canEnableDisableCalls` in `channel_info/index.ts`, so that we don't get to this point when calls aren't enabled (`canEnableDisableCalls` will be false in `channel_info.tsx`).

Reply:
Thanks. Yeah, wasn't exactly sure myself at which level to add these. I'll review.